service/routes.py

```diff
@@ -48,24 +48,24 @@ def index():
 
 
 ######################################################################
-# LIST ALL PETS
+# LIST ALL EXISTING ORDERS
```

Comment:
This code looks for categories first -- since we just want to call all orders, can make orders var = all() (which calls the all() instance from main model) 

Just remove the unnecessary code so looks like below (tested). 

`@app.route("/orders", methods=["GET"])
def list_orders():
    """ Returns all of the Orders """
    app.logger.info("Request for Order list")
    orders = Order.all()
    results = [order.serialize() for order in orders]
    return make_response(jsonify(results), status.HTTP_200_OK)`